lib/verify.go

```diff
@@ -130,8 +131,17 @@ func VerifyChain(certs []*x509.Certificate, ocspStaple []byte, dnsName, caPath s
 		result.Error = fmt.Sprintf("%s", err)
 		return result
 	}
+	// expectedName could be a hostname or could be a SPIFFE ID (spiffe://...)
+	// x509 package doesn't support verifying SPIFFE IDs. We thus have to do a bit more work here.
+	spiffeIDExpected := strings.HasPrefix(expectedName, "spiffe://")
+	var expectedDnsName string
```

Comment:
nit: this should be `expectedDNSName`, since Go likes keeping acronym casing the same (`DNS` instead of `Dns`)

Reply:
Done!